core/utils.class.inc.php

```diff
@@ -439,6 +439,7 @@ static public function DoPostRequest($sUrl, $aData, $sOptionnalHeaders = null, &
 
 	/**
 	 * Pretty print a JSON formatted string. Copied/pasted from http://stackoverflow.com/questions/6054033/pretty-printing-json-with-php
+	 * @deprecated 1.3.0
```

Comment:
When possible it's nice to explain what to use instead O:)

```suggestion
	 * @deprecated 1.3.0 use `json_encode($json, JSON_PRETTY_PRINT);` instead
```